app/src/main/java/io/homeassistant/companion/android/webview/WebViewActivity.kt

```diff
@@ -279,6 +291,79 @@ class WebViewActivity : AppCompatActivity(), io.homeassistant.companion.android.
                             "config_screen/show" -> startActivity(
                                 SettingsActivity.newInstance(this@WebViewActivity)
                             )
+                            "panel/actions" -> {
+                                if (Build.VERSION.SDK_INT >= 26) {
+                                    val shortcutManager =
+                                        getSystemService(ShortcutManager::class.java)
+                                    if (shortcutManager!!.isRequestPinShortcutSupported) {
+                                        val panel: Panel = Gson().fromJson(
+                                            json.getJSONObject("payload").toString(),
+                                            Panel::class.java
+                                        )
+                                        val title: String? = when (panel.title) {
+                                            "calendar" -> getString(R.string.calendar)
+                                            "config" -> getString(R.string.config)
+                                            "developer_tools" -> getString(R.string.developer_tools)
+                                            "history" -> getString(R.string.history)
+                                            "logbook" -> getString(R.string.logbook)
+                                            "mailbox" -> getString(R.string.mailbox)
+                                            "map" -> getString(R.string.map)
+                                            "profile" -> getString(R.string.profile)
+                                            "shopping_list" -> getString(R.string.shopping_list)
+                                            "states" -> getString(R.string.states)
+                                            else -> panel.title
```

Comment:
Maybe we just capitalize and replace `_` with ` ` rather than translate?

Reply:
Not possible since these names (Overview is not states) aren't correct and of course won't be translated. This follows the frontend's way of using panels